src/cnxml/deserialize.js

```diff
@@ -155,6 +156,22 @@ function admonition(el, next) {
 }
 
 
+/**
+ * Process data for emphasis marks.
+ */
+function code(el, next) {
+    let object = 'block'
+    if (el.parentNode && el.parentNode.tagName.toLowerCase() === 'para') {
```

Comment:
Inline code fragments can be placed in more than just paragraphs (e.g. figure captions).

Reply:
Since the `display` attribute is required for block code fragments, we could just case over `getAttribute('display') || 'inline'`.